pkg/scheduler/util.go

```diff
@@ -60,16 +58,41 @@ func loadSchedulerConf(confStr string) ([]framework.Action, []conf.Tier, error)
 		}
 	}
 
-	actionNames := strings.Split(schedulerConf.Actions, ",")
-	for _, actionName := range actionNames {
-		if action, found := framework.GetAction(strings.TrimSpace(actionName)); found {
-			actions = append(actions, action)
-		} else {
-			return nil, nil, fmt.Errorf("failed to found Action %s, ignore it", actionName)
+	for i, action := range schedulerConf.Actions {
+		if _, found := framework.GetAction(action.Name); !found {
+			return nil, fmt.Errorf("failed to found Action %s, ignore it", action.Name)
+		}
+
+		if action.Options == nil {
+			schedulerConf.Actions[i].Options = map[string]string{}
```

Comment:
let's move options to each action accordingly.

Reply:
action.Options means "the Options of this action"